mpy-cross/main.c

```diff
@@ -322,8 +344,8 @@ MP_NOINLINE int main_(int argc, char **argv) {
     }
 
     if (input_file == NULL) {
-        mp_printf(&mp_stderr_print, "no input file\n");
-        exit(1);
+        // default to stdin
+        input_file = "-";
```

Comment:
I prefer the existing behaviour of giving an error if no files are specified (printing "no input file").  Because that matches gcc/clang, and they are the closest unix tool to mpy-cross.

Reply:
Done.